src/components/views/rooms/ReadReceiptMarker.js

```diff
@@ -155,7 +155,13 @@ export default class ReadReceiptMarker extends React.Component {
 
         // then shift to the rightmost column,
         // and then it will drop down to its resting position
-        startStyles.push({ top: startTopOffset+'px', left: '0px' });
+        //
+        // XXX: We use a `left: 1px` to trick velocity-animate into actually animating. This
+        // is a very annoying bug where if it thinks there's no change to `left` then it'll
+        // skip applying it, thus making our read receipt at +14px instead of +0px like it
+        // should be. This does cause 1px of drift for read receipts, however nobody should
+        // notice this while it's also falling.
+        startStyles.push({ top: startTopOffset+'px', left: '1px' });
```

Comment:
can you do something naughty like switch from 0px to 0em to avoid the drift? or use a small fractional offset, in order to remain pixel perfect?

Reply:
it parses the numbers internally, so `0em` is effectively the same as `0px` (it determines change of value over time, not units, annoyingly). 

`0.001px` seems to be the smallest we can go - will use that.